.changeset/neat-papayas-brake.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': patch
+---
+
+Fixes an issue where custom `assetFileNames` were not working as expected.
```

Comment:
Can you provide a better description, please? "not working as expected" is a bit broad. You provided a good description in the PR, and maybe some of that description could be moved here. 

For example, we should tell the users in which case the `assetFileNames` were not working (when read from the client?).

Reply:
Thanks for the feedback. I updated it.